File: mRemoteV1/CS/Connection/Connection.Protocol.Rlogin.cs
using System.Collections.Generic;
using System;
using AxWFICALib;
using System.Drawing;
using System.Diagnostics;
using System.Data;
using AxMSTSCLib;
using Microsoft.VisualBasic;
using System.Collections;
using System.Windows.Forms;


namespace mRemoteNG.Connection.Protocol
{
	public class Rlogin : PuttyBase
	{
				
		public Rlogin()
		{
			this.PuttyProtocol = Putty_Protocol.rlogin;
		}
				
		public enum Defaults
		{
			Port = 513
		}
	}
}

File: mRemoteV1/Connection/Connection.Protocol.Rlogin.vb
Namespace Connection

    Namespace Protocol
        Public Class Rlogin
            Inherits PuttyBase

            Public Sub New()
                Me.PuttyProtocol = Putty_Protocol.rlogin
            End Sub

            Public Enum Defaults
                None = 0
                Port = 513
            End Enum
        End Class
    End Namespace

End Namespace